index.html

```diff
@@ -0,0 +1,147 @@
+          <ul>
+              <li><a href=""><i class="fab fa-facebook-f"></i></a></li>
+              <li><a href=""><i class="fab fa-twitter"></i></a></li>
+              <li><a href="">English</a></li>
+              <li><a href=""> My Page</a></li>
+          </ul>
+      </div>
+  </div>
+  <div class="desktop-navbar">
+      <div class="logo">
+          <img src="./images/riana.png" alt="logo-image">
+      </div>
+      <div class="desktop-nav-list">
+          <ul>
+          
+              <li><a href="about.html">About</a></li>
+              <li><a href="#team-id">program</a></li>
+              <li><a href="#team-id"> Sponsor</a></li>
+              <li><a href="#team-id"> News</a></li>
+          </ul>
+      </div>
+  </div>
+  </nav>
+  <div class="welcome">
+    <h3 class="text">"Hello how are you!"</h3>
+    <h1 class="text">Concert King's Jimin</h1>
+  </div>
+  <div class="welcome-description">
+    <p class="welcome-text">
+       Stage King' Jimin wows fans and netizens with his exhilarating dance,
+       vocals and visuals after BTS's Busan concert
+    </p>
+  </div>
+  <div class="event-date">
+    <h2 class="fan-text">Be fan now</h2>
+    <p class="fan-p">@ BTS, The world rocking <strong>boys</strong></p>
+  </div>
+</div>
+</section>
+    </header>
+        <section class="program">
+          <h2 id="services-title">Main songs on stage</h2>
+          <div id="redline"></div>
+          <div class="service-cards">
+          <div class="card">
+              <img src="./images/permission.png" class="card-images" alt="">
+              <h3 id="program-title"> Permission </h3>
+              <p> you can watch the recorded concert on bts main channel.</p>
+          </div>
+          <div class="card">
+              <img src="./images/fire.png" class="card-images" alt="">
+              <h3 id="program-title"> fire</h3>
+              <p>you can watch the recorded concert on bts main channel.</p>
+          </div>
+          <div class="card">
+              <img src="./images/dynimate.png" class="card-images" alt="">
+              <h3 id="program-title"> Dynamite</h3>
+              <p>you can watch the recorded concert on bts main channel.</p>
+          </div>
+          <div class="card">
+              <img src="./images/butter.png" class="card-images" alt="">
+              <h3 id="program-title"> Butter </h3>
+              <p> you can watch the recorded concert on bts main channel.</p>
+          </div>
+          <div class="card">
+              <img src="./images/fake love .png" class="card-images" alt="">
+              <h3 id="program-title"> Fake love</h3>
+              <p>  you can watch the recorded concert on bts main channel.</p>
+          </div>
+      </div>
+      <div id="view-prog-div">
+          <a id="view-program" href=""> To View full concert </a>
+      </div>
+      <div class="join-bt">
+        <a id="join" href="">Join bts 2022</a>
```

Comment:
- [ ] As requested by the previous reviewer, This button on mobile version still does not fully match the provided Behance design. Kindly look into it, it is remaining with a very small bit to be almost perfect.

**This is how I am seeing it right now:**

![image](https://user-images.githubusercontent.com/86318284/198078228-5170427d-2165-413b-9f01-7b6d83e207aa.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/198078334-c9ec078a-e9f2-403a-afa2-64a4e78b1462.png)